### PR TITLE
add contribution notes for designers to `contributing.md`

### PR DESCRIPTION
**Pull request by brandonhaslegs**
Just adding some notes for how to contribute as a designer. Open to any suggestions to make this more clear or specific. We haven't actually accepted any open source design contributions yet so it's a totally new process.

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -64,6 +64,14 @@ We're writing documentation as we are developing new features. If you find
 something that is confusing or not covered at all, feel free to
 [open a bug][ob] or [contribute][cd]
 
+## Contributing to the Design
+
+Since Radicle is an open source project, anyone can contribute. This is normal in open source development, but we do it for design too! Here's how you can get started:
+
+I’d suggest starting by joining our [Discord server](ds) to chat with me or anyone on the team and ask any questions you have. It’s all public and open for anyone to join and chat in. We even have our “internal” chats public on there where we chat all day about features. The #Upstream channel is an example of that.
```
Comment (rudolfs) [inline docs/contributing.md:71]: Links have to be formatted like this, for them to render correctly: `[Discord server][ds]`.

File: docs/contributing.md [hunk 2]
```diff
@@ -64,6 +64,14 @@ We're writing documentation as we are developing new features. If you find
 something that is confusing or not covered at all, feel free to
 [open a bug][ob] or [contribute][cd]
 
+## Contributing to the Design
+
+Since Radicle is an open source project, anyone can contribute. This is normal in open source development, but we do it for design too! Here's how you can get started:
```
Comment (rudolfs) [inline docs/contributing.md:69]: Could you hard-wrap the lines around 80 characters?

Comment (brandonhaslegs) [reply docs/contributing.md:69]: Is there a VS Code plugin for this?

File: docs/contributing.md [hunk 3]
```diff
@@ -82,3 +102,8 @@ something that is confusing or not covered at all, feel free to
 [rd]: https://docs.radicle.xyz
 [rt]: https://github.com/radicle-dev/radicle-upstream/issues/1958
 [up]: http://docs.radicle.xyz/docs/using-radicle/creating-patches
+[ds]: https://discord.gg/HRdnwAwGbG
```
Comment (rudolfs) [inline docs/contributing.md:105]: This shows me an invite when I click it. Should it not point to the channel?
Like: https://discord.com/channels/841318878125490186/843873418205331506.

File: docs/contributing.md [hunk 4]
```diff
@@ -82,3 +102,8 @@ something that is confusing or not covered at all, feel free to
...
+[dn]: https://github.com/radicle-dev/radicle-upstream/issues?q=is%3Aopen+is%3Aissue+label%3Adesign-needed
+[ff]: https://www.figma.com/file/tVqxYmi0YM4TRRebCXWZaF/Radicle-Funding?node-id=393%3A8210
```
Comment (rudolfs) [inline docs/contributing.md:107]: This opens Figma on your playground. Is that correct?
Screenshot 2021-09-23 at 12 53 06

File: docs/contributing.md [hunk 5]
```diff
@@ -64,6 +64,26 @@ We're writing documentation as we are developing new features. If you find
 something that is confusing or not covered at all, feel free to
 [open a bug][ob] or [contribute][cd]
 
+## Contributing to the Design
+
+Since Radicle is an open source project, anyone can contribute. This is normal
+in open source development, but we do it for design too! Here's how you can get started:
```
Comment (rudolfs) [inline docs/contributing.md:70]: This still wraps around 88. Could you configure your editor to wrap at 80?

File: docs/contributing.md (hunk 4 above)
Comment (brandonhaslegs) [reply docs/contributing.md]: nope fixed

File: docs/contributing.md (hunk 3 above)
Comment (brandonhaslegs) [reply docs/contributing.md]: 🤷‍♂️ That's what we link to from our website?

File: docs/contributing.md [hunk 6]
```diff
@@ -47,38 +47,64 @@ feat(proxy): improve session (#380)
...
+https://github.com/radicle-dev/radicle-docs/issues [pr]:
+https://github.com/radicle-dev/radicle-upstream/pulls [ra]: https://radicle.xyz
+[rd]: https://docs.radicle.xyz [rt]:
+https://github.com/radicle-dev/radicle-upstream/issues/1958 [up]:
+http://docs.radicle.xyz/docs/using-radicle/creating-patches [ds]:
+https://discord.gg/HRdnwAwGbG [dn]:
+https://github.com/radicle-dev/radicle-upstream/issues?q=is%3Aopen+is%3Aissue+label%3Adesign-needed
+[ff]:
+https://www.figma.com/file/tVqxYmi0YM4TRRebCXWZaF/Radicle-Funding?node-id=393%3A8210
+[bo]: https://github.com/brandonhaslegs [jd]: https://github.com/juliendonck
```
Comment (rudolfs) [inline docs/contributing.md:110]: This part shouldn't be wrapped.

Comment (brandonhaslegs) [reply docs/contributing.md:110]: 🙄

File: docs/contributing.md (hunk 3 above)
Comment (brandonhaslegs) [reply docs/contributing.md:108]: I will make another issue for this btw.

File: docs/contributing.md (hunk 4 above)
Comment (rudolfs) [reply docs/contributing.md:110]: For some reason it still opens it for me on your playground. 🤷

File: docs/contributing.md [hunk 7]
```diff
@@ -47,30 +47,53 @@ feat(proxy): improve session (#380)
...
+  You’ll need to duplicate the file and make changes in your own private file.
+  If we accept them, we’ll integrate them into the official file.
+- If you want to submit a design solution on GitHub, just post screenshots of
+  your solution and a description on the issue (see here for an example of this
+  in action)
 
 
 [ar]: http://docs.radicle.xyz/docs/using-radicle/tracking-and-viewing#adding-remotes
-[cb]: https://docs.radicle.xyz/docs/using-radicle/overview
+[cb]: https://docs.radicle.xyz/docs/using-radicle/overview
```
Comment (rudolfs) [inline docs/contributing.md:93]: Could you maybe have a look at your editor settings to make sure you don't commit trailing whitespace?

File: docs/contributing.md (hunk 3 above)
Comment (rudolfs) [reply docs/contributing.md:108]: I think you mixed up the links. You can delete this line and update the link in the text to `[Discord server][dc]`.

**Review by rudolfs — APPROVED** [review]
🎉

**Pull request merged**